layers/+distributions/spacemacs-base/keybindings.el

```diff
@@ -584,10 +584,10 @@ otherwise it is scaled down."
 
 ;; Transparency transient-state
 
-(defun spacemacs/toggle-transparency ()
-  "Toggle between transparent or opaque display."
+(defun spacemacs/toggle-transparency (&optional frame)
+  "Toggle between transparent or opaque display for given or current frame."
   (interactive)
-  (let* ((frame (selected-frame))
+  (let* ((frame (or frame (selected-frame)))
```

Comment:
If `frame` is nil, `frame-parameter` defaults to the selected frame, so we don't need to explicitly set `frame`.